src/Api.ts

```diff
@@ -38,6 +38,10 @@ class Api {
     return this.axios.get('/app/preferences');
   }
 
+  public shutdownApplication() {
+    return this.axios.get('/app/shutdown');
```

Comment:
Should be post?

Reply:
I don't see any specification in the [docs](https://github.com/qbittorrent/qBittorrent/wiki/WebUI-API-(qBittorrent-4.1)#shutdown-application), GET works without any problem